pyethapp/config.py

```diff
@@ -103,13 +115,14 @@ def _fix_accounts(path):
         write_config(config)
 
 
-def load_config(path=default_config_path):
+def load_config(path=default_config_path, fix_accounts=True):
```

Comment:
Any particular action (any write?) should happen on data dir or is enough to check data dir after `pyethapp config` ?